### DISCUSSION TITLE
Considerations for alternative keyboards such as the Corne and Gergoplex

### DISCUSSION THREAD

**Discussion by jtrv** [binarybottle/engram-layout#31] [Ideas]
I am considering using this layout with a more minimal ortholinear design such as the corne or gergoplex. I was going to try and adapt a layout from feelings and assumptions after trial and error once I create one and test it with engram. However I thought I should reach out and see if anybody has already considered applying the methodology to alternative smaller layouts to maximize the ergonomics of the additional layers. 

Also has anybody made considerations for modifier keys or even key-down/up events. I.E. using [xcape](https://github.com/alols/xcape#xcape) (to escape when `ctrl` is tapped and act as ctrl when held down) these could double up thumb keys ie
```
Left Thumbs     
 _____________   _______________   _____________
| lower-layer | | mod/backspace | | ctrl/escape |
 -------------   ---------------   -------------

Right Thumbs
 _____________   _________   _____________
| shift/space | | alt/tab | | upper-layer |
 -------------   ---------   -------------
```

**Comment by binarybottle** [comment]
In looking at these minimal ortholinear layouts, they appear to have three by five or six keys for each hand, so more than enough for all of the letters and punctuation. Is your question how to access the number and symbol-above-the-number keys?

**Comment by binarybottle** [comment]
Don't these ortholinear keyboards have 3 x 5 or 6 keys, which more than accommodate the 26 letters and common punctuation? Are you asking about how to accommodate numbers and the symbols above the numbers?

**Reply by jtrv** [reply]
Yes that was what I was curious about. Since you can arrange the `numbers` / `symbols-above-numbers` together on their own third layer you can utilize home-row ergonomics and placements that are normally occupied by letters. So I was curious as to how you might place those to maximize ergonomics.

**Reply by jtrv** [reply]
or if you might even shift some punctuation into the third layer instead to take advantage of those home-row placements

**Comment by binarybottle** [comment]
Great questions! While you can move everything around wherever you would like to optimize to your own personal taste, there are advantages to keeping consistent with the symmetry in the full layout. I would try something simple at first, like try accessing the numbers and the symbols above the numbers from the home row and above the home row with a trigger key, like AltGr.

**Reply by jtrv** [reply]
Oh ok thank you I appreciate your help!

**Comment by binarybottle** [comment]
Please see https://github.com/binarybottle/engram-2021/discussions/86#discussion-10126586 for an announcement of the new Engram approach/layouts/repo!
